images/router/haproxy/conf/haproxy-config.template

```diff
@@ -164,15 +164,7 @@ frontend public
   acl secure_redirect base,map_reg(/var/lib/haproxy/conf/os_route_http_redirect.map) -m found
   redirect scheme https if secure_redirect
 
-  # Check if it is an edge or reencrypt route exposed insecurely.
-  acl route_http_expose base,map_reg(/var/lib/haproxy/conf/os_route_http_expose.map) -m found
-  use_backend %[base,map_reg(/var/lib/haproxy/conf/os_route_http_expose.map)] if route_http_expose
-
-  # map to http backend
-  # Search from most specific to general path (host case).
-  # Note: If no match, haproxy uses the default_backend, no other
-  #       use_backend directives below this will be processed.
-  use_backend be_http:%[base,map_reg(/var/lib/haproxy/conf/os_http_be.map)]
+  use_backend %[base,map_reg(/var/lib/haproxy/conf/os_http_be.map)]
```

Comment:
Any reason why you only added `be_http:` and `be_edge_http:` to the map backend entry value and not for the redirect and tcp maps? It makes it inconsistent.

Reply:
I left out the redirects because that is a specific operation that does not link to a use_backend statement. It is determining to allow redirect based on if there is an entry in the map.

Reply:
No worries - I was just asking. I've made the change in my branch to make it consistent as part of the dynamic config changes anyway so its cool.